js/main.js

```diff
@@ -22,13 +22,13 @@ form.addEventListener('submit', function (event) {
     ul.prepend(renderEntry(submitObj));
   }
   if (data.editing !== null) {
-    const li = document.querySelectorAll('li');
     const updatedForm = {
       $title: form.elements.title.value,
       $url: form.elements.url.value,
       $notes: form.elements.notes.value
     };
     updatedForm.entryId = data.editing.entryId;
+    const li = document.querySelectorAll('li');
     for (let i = 0; i < data.entries.length; i++) {
```

Comment:
after this for loop (on line 40), add `data.editing = null;`
This will clear out the stored values in your data object so that when you attempt to edit another entry it will work properly. This is for a similar reason I mention on line 177.